<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSetting</class>
 <widget class="QDialog" name="AdvancedSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Advanced Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="aiSettingGroupBox">
     <property name="title">
      <string>AI HyperParameter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="as0_info">
        <property name="text">
         <string>가능하면 건들지 않는 것이 좋습니다.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as1_imageDir" stretch="1,4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Image Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imageDirEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as2_input" stretch="1,4">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Input (List)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as3_output" stretch="1,4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Output (List)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as4_sequenceLength" stretch="1,4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Sequence Length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sequenceLengthEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as5_units" stretch="1,4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Units</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="unitsEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as6_epochs" stretch="1,4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Epochs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="epochsEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as7_batchSize" stretch="1,4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Batch Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="batchSizeEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as8_outlierCheck">
        <item>
         <widget class="QCheckBox" name="outlierCheckBox">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Outlier Check?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as9_outlierColumn" stretch="1,4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Outlier Column</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outlierColumnEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as10_outlierWeight" stretch="1,4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Outlier Weight (IQR)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outlierWeightEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="as11_verbose" stretch="1,4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Verbose</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="verboseEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdvancedSetting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdvancedSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
